README.md

```diff
@@ -1,6 +1,13 @@
 # OpenFisca-France
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-france.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-france)
+[![Newsletter](https://img.shields.io/badge/newsletter-subscribe!-informational.svg?style=flat)](mailto:contact%40openfisca.org?subject=Subscribe%20to%20your%20newsletter%20%7C%20S'inscrire%20%C3%A0%20votre%20newsletter&body=%5BEnglish%20version%20below%5D%0A%0ABonjour%2C%0A%0AVotre%C2%A0pr%C3%A9sence%C2%A0ici%C2%A0nous%C2%A0ravit%C2%A0!%20%F0%9F%98%83%0A%0AEnvoyez-nous%20cet%20email%20pour%20que%20l'on%20puisse%20vous%20inscrire%20%C3%A0%20la%20newsletter.%20%0A%0AAh%C2%A0!%20Et%20si%20vous%20pouviez%20remplir%20ce%20petit%20questionnaire%2C%20%C3%A7a%20serait%20encore%20mieux%C2%A0!%0Ahttps%3A%2F%2Fgoo.gl%2Fforms%2F45M0VR1TYKD1RGzX2%0A%0AAmiti%C3%A9%2C%0AL%E2%80%99%C3%A9quipe%20OpenFisca%0A%0A%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%20ENGLISH%20VERSION%20%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%3D%0A%0AHi%2C%20%0A%0AWe're%20glad%20to%20see%20you%20here!%20%F0%9F%98%83%0A%0APlease%20send%20us%20this%20email%2C%20so%20we%20can%20subscribe%20you%20to%20the%20newsletter.%0A%0AAlso%2C%20if%20you%20can%20fill%20out%20this%20short%20survey%2C%20even%20better!%0Ahttps%3A%2F%2Fgoo.gl%2Fforms%2FsOg8K1abhhm441LG2%0A%0ACheers%2C%0AThe%20OpenFisca%20Team)
```

Comment:
Because I do not natively read urlencoded, I do not particularly relish reviewing this, nor do I look forward to changing it in the future. How would you go about it if our roles were swapped?

Reply:
1. Je cliquerais le bouton : ![capture d ecran 2019-02-20 a 10 38 16](https://user-images.githubusercontent.com/329236/53081600-77a13500-34fb-11e9-9027-b3d7289592e1.png)

2. Je cliquerait sur le badge : ![capture d ecran 2019-02-20 a 10 38 35](https://user-images.githubusercontent.com/329236/53081622-82f46080-34fb-11e9-94d6-7b8d8b84221a.png)

3. Je m'assurerait que le subject et le message de l'email me conviennent :

<img width="722" alt="capture d ecran 2019-02-20 a 10 38 45" src="https://user-images.githubusercontent.com/329236/53081735-ba630d00-34fb-11e9-8bd5-53600155f0fb.png">